GraphSturcture/src/graphStructure/Vertex.java

```diff
@@ -0,0 +1,79 @@
+package graphStructure;
+
+import java.util.Set;
+import java.util.Map;
+import java.util.HashMap;
+import java.util.Iterator;
+
+public class Vertex {
+
+  private Integer vertexId;
+  private Map adjVertices;
```

Comment:
You only need to maintain a map of edges. Also, you need to specify the type of the map while define it. You'll see, that will greatly simplify your code in Graph (remove a vertex or an edge for example).